styles/variables-flatly.less

```diff
@@ -0,0 +1,80 @@
+/* Basic page-wide colors */
```

Comment:
Needed for test removed errorneously in `b5d444eca6d7d43eda77a7fb0e985c1476d4f1dc`